stories/Modals.stories.tsx

```diff
@@ -34,3 +35,26 @@ std.story = {
   name: 'Search',
   parameters: { note: 'search results modal' }
 }
+
+export const newDataset = () => {
+  return (
+    <div style={{ margin: 0, padding: 30, height: '100%', background: '#F5F7FA' }}>
+      <div style={{ width: 800, margin: '2em auto' }}>
+        <Router>
+          <Route render={(props) =>
+            <NewDatasetComponent
+              {...props}
+              modal={{ type: ModalType.NewDataset }}
+              onDismissed={() => console.log('New dataset modal is dismissed.')}
+            />}
+          />
+        </Router>
+      </div>
+    </div>
+  )
+}
```

Comment:
whenever you're using a direct return, drop the braces & the `return`:
```suggestion
export const newDataset = () => (
  <div style={{ margin: 0, padding: 30, height: '100%', background: '#F5F7FA' }}>
    <div style={{ width: 800, margin: '2em auto' }}>
      <Router>
        <Route render={(props) =>
          <NewDatasetComponent
            {...props}
            modal={{ type: ModalType.NewDataset }}
            onDismissed={() => console.log('New dataset modal is dismissed.')}
          />}
        />
      </Router>
    </div>
  </div>
)
```